### ISSUE TITLE
Performance after quantization

### ISSUE THREAD

**Issue by Zaman32** [Xilinx/Vitis-AI#744]
Hello, 

I have been trying to accelerate a UNET model for semantic segmentation. But after quantization its performance decreases significantly. It barely detects any class. I tried the finetuning, after quantization, approaches mentioned in the documentation but in does not improves the performance. Could you suggest me some approach to improve the model performance after quantization that really works?

**Comment by qianglin-xlnx** [comment]
Hi @Zaman32 
When you say the performance decreases after quantization and it barely detects any class, do you mean the accuracy of the quantized model is lower than the float model? How much lower is it?
And do you run the quantized model on the target or host? For target, which platform do you use?
The UNET model you used is based on Pytorch framework?

**Comment by qianglin-xlnx** [comment]
Hi @Zaman32 
Has this issue been solved? If not, could you please provide more information. Thank you.

**Comment by niuxjxlnx** [comment]
Please refer to UNET models deployment in vitis-ai model zoo.
https://github.com/Xilinx/Vitis-AI/tree/master/model_zoo
